docs/behave.rst

```diff
@@ -55,6 +55,11 @@ You may see the same information presented below at any time using ``behave
 
     Directory in which to store JUnit reports.
 
+.. option:: --runner-class
+
+    Tells Behave to use a specific runner. The default is 
+    ``behave.runner.Runner``.
```

Comment:
I'm not 100% sure this is the syntax. @bittner  ?

Reply:
Looks good. Simply make sure it looks like the other examples. :smile_cat: 

Maybe add, "This allows you to use your own custom runner." – That would explain the motivation for this option, wouldn't it? (Or maybe not; it would be less concise then, wouldn't it?)

Reply:
Generally, in commit messages, try to follow the [seven rules for a great commit message](https://chris.beams.io/posts/git-commit/#seven-rules). Especially rule number 5, e.g. 

> Add ability to use custom runner to the behave command

> Add docs for --runner-class option

:grimacing:

Reply:
You can run `tox -e docs` to build the documentation locally. (Not sure on your Mac, but on my awesome Ubuntu box it works! :smirk:)

It's a good idea to run this command twice (to let it resolve document dependencies). Then run `open .tox/docs/tmp/html/en/index.html` to take a look at it in your Web browser. Looks a bit different than on RTD, don't worry, but the content should be there.